src/www/ui/api/Helper/UploadHelper.php

```diff
@@ -643,10 +669,10 @@ public function getUploadLicenseList($uploadId, $agents, $printContainers, $bool
     $restHelper = $container->get('helper.restHelper');
     $uploadDao = $restHelper->getUploadDao();
     $agentDao = $container->get('dao.agent');
-
+    $dbManager = $restHelper->getDbHelper()->getDbManager();
     $uploadTreeTableName = $uploadDao->getUploadtreeTableName($uploadId);
     $parent = $uploadDao->getParentItemBounds($uploadId, $uploadTreeTableName);
-
+    $groupId = Auth::getGroupId();
```

Comment:
```suggestion
    $groupId = $restHelper->getGroupId();
```

Reply:
This is causing 500 internal server error